docs/wip/data-fetching.md

```diff
@@ -1,8 +1,45 @@
-## Gatsby config queries
-
 ## How to fetch data for your pages
 
+One of the main benefits of Gatsby is its hybrid nature capable of static and dynamic data fetching, which means that is possible to fetch data in build time as well in runtime on client-side. The choice of which approach to use depends on the nature of the data and its context, in the section below we will explain the difference with examples.
+
+> If you want to read more about this topic, check the [Data Fetching](https://www.gatsbyjs.com/docs/data-fetching/) Gatsby documentation.
+
 ### Static data
+
+Static data is the content that does not change frequently or per user interaction. For example, a product's page can be a data retrieve static since the description and attributes of product don't change very often. This means that this data can be fetch during build time and when the user access the products page the data is already loaded.
+
+There is two ways to fetch static data in Gatsby: [Page Query](https://www.gatsbyjs.com/docs/recipes/querying-data#querying-data-with-a-page-query) and [Static Query](https://www.gatsbyjs.com/docs/how-to/querying-data/static-query/#reach-skip-nav).
+
 ### Dynamic data
 
-### What we provided out of the box with `gatsby-source-vtex`
+If the data constantly changes or depends on the user's interaction, it is a strong indication that it must be dynamic. For example, the shipping calculation changes depending on the location of the user who is interacting or other example is the implementation of a shopping cart that changes its data every time a product is added, being necessary to communicate with the backend of the application every time that the user interacts with the cart.
```

Comment:
"If the data constantly changes or depends on the user's interaction,  make it to be dynamic fetched. 

For example: when a shipping calculation changes depending on the user's location or when implementing a shopping cart that display diferrent content every time a product is added, demanding backend communcation at every user's change — use the dynamic"